tasks/strings/strings.js

```diff
@@ -4,23 +4,38 @@
  * @param {string} str
  */
 function reverseString(str) {
+  if (typeof (str) == 'number') return 'This is not a string!';
```

Comment:
it is better to check in a more wide-spread way: type of str !== 'string'

Reply:
fixed